content/blog/findings-when-coverting-tests-to-pester5.md

```diff
@@ -0,0 +1,303 @@
+---
+title: "Findings when converting tests to Pester 5"
```

Comment:
I'd rename that article.
"Converting to Pester 5 from the trenches" something like this.